src/Catty/PocketPaint/ViewController/PaintViewController.m

```diff
@@ -183,6 +183,8 @@ - (void)setupCanvas
     NSInteger height = (NSInteger)self.projectHeight;
     CGRect rect = CGRectMake(0, 0, width, height);
     self.drawView = [[UIImageView alloc] initWithFrame:rect];
+    self.drawView.accessibilityIdentifier = @"Canvas";
```

Comment:
Why do we need this here? Is this ticket related to Paint?

Reply:
For creating a new look, I used 'Draw new image' option. When doing that, I was required to make changes on the drawView in order to save it. So I had to tap drawView. So this is to identify drawView from that test.

Reply:
That's cool! Could you here please also include the "Paint" keyword so that this identifier will be more readable from "outside"? (e.g. `PaintCanvas` or similar).